documentation/book/proc-configuring-clients-to-trust-cluster-ca.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+= Configuring clients to trust the Cluster CA 
+
+If a Kafka client wants to connect to the `tls` listener on port 9093, it needs to trust the cluster CA certificate in order to verify the broker certificate received during the SSL/TLS handshake.
+
+.Procedure
+
+. The cluster CA certificate can be extracted from the generated `<cluster-name>-cluster-ca-cert` `Secret`.
++
+ifdef::Kubernetes[]
+On {KubernetesName}, the certificate can be extracted with the following command:
++
+[source,shell]
+kubectl get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
+endif::Kubernetes[]
+On {OpenShiftName}, the certificate can be extracted with the following command:
++
+[source,shell]
+oc get secret <cluster-name>-cluster-ca-cert -o jsonpath='{.data.ca\.crt}' | base64 -d > ca.crt
```

Comment:
Should we use `oc extract` here? It think its easier to use and should work also on non-Linux OSes.